### PR TITLE
Added proper i128 support

### PR DESCRIPTION
**Pull request by zesterer**
# Description of change

Added `autocfg` to build dependencies to detect support for i128, and produce an feature to indicate this. `autocfg` works by actually testing small code snippets to determine support.

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix

## How the change has been tested

This hasn't been tested yet, I'm in the process of doing so.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have updated the CHANGELOG.md, if my changes are significant enough

### CODE REVIEW

File: bee-ternary/CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 0.3.4-alpha - 2020-11-13
+
+### Added
+
+- Added proper `i128`/`u128` support detection
```
Comment (thibault-martinez) [inline bee-ternary/CHANGELOG.md]:
```suggestion
- Added proper `i128`/`u128` support detection.
```

File: bee-ternary/build.rs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+fn main() {
```
Comment (thibault-martinez) [inline bee-ternary/build.rs:12]: Copyright header

File: bee-ternary/CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 0.3.4-alpha - 2020-11-13
```
Comment (thibault-martinez) [inline bee-ternary/CHANGELOG.md:22]: You did not update the version in Cargo.toml ?

**Review by thibault-martinez — CHANGES_REQUESTED** [review]

File: bee-ternary/CHANGELOG.md (hunk 3 above)
Comment (zesterer) [reply bee-ternary/CHANGELOG.md:22]: Oops!

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**